packages/client3/constants/metadata/types.ts

```diff
@@ -1,4 +1,5 @@
 export interface PoolMetadata {
+  artist: string;
   name: string;
```

Comment:
Sorry, what is the difference between `artist` and `name`?

Reply:
this is gonna change when I merge the code from goldfinch, but to clarify: name here is the pool name and artist is the artist name, when I merge the code from goldfinch I should abstract the artist from another metadata somehow